po/r-novice-gapminder.ja.po

```diff
@@ -18573,7 +18549,7 @@ msgstr ""
 # header
 #: r-novice-gapminder/_episodes/14-tidyr.md:303
 msgid "## From long to intermediate format with spread()"
-msgstr "## From long to intermediate format with spread()"
+msgstr "## spread() で縦長から中間形式へ"
```

Comment:
I think they change from spread() to pivot_wider().

Reply:
leave as-is for now